config/application.rb

```diff
@@ -45,6 +46,11 @@ class Application < Rails::Application
 
     config.label_templates = Rails.application.config_for(:label_templates)
 
+    config.eager_load_paths += %W[#{config.root}/app/lib/utils #{config.root}/app/lib/validators
```

Comment:
Are these required because their namespacing isn't right and zeitwerk can't find it properly?

Reply:
Yes. Labwhere has some strange folder structure. As per above I wanted to stop somewhere. I will probably look at these again when I update some of the other dependencies.